sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolicExecution/ExplodedGraph.cs

```diff
@@ -326,7 +326,7 @@ private void EnqueueStartNode()
             var initialProgramState = new ProgramState();
             foreach (var parameter in declarationParameters)
             {
-                var sv = new SymbolicValue();
+                var sv = SymbolicValue.Create(parameter.Type);

```

Comment:
But if I am using the factory everywhere I kindda imply that I am handling this case when there is a nullable value whereas I don't. For now I am only handling identifiers and method parameters.